bullet_train-api/app/views/account/platform/access_tokens/edit.html.erb

```diff
@@ -1,12 +1,7 @@
-<%= render 'account/shared/page' do |p| %>
-  <% p.content_for :title, t('.section') %>
-  <% p.content_for :body do %>
-    <%= render 'account/shared/box', divider: true do |p| %>
-      <% p.content_for :title, t('.header') %>
-      <% p.content_for :description, t('.description') %>
-      <% p.content_for :body do %>
-        <%= render 'form', access_token: @access_token %>
-      <% end %>
-    <% end %>
+<%= render 'account/shared/page' do |page| %>
+  <% page.title t('.section') %>
+  <% page.body.render 'account/shared/box', divider: true do |box| %>
```

Comment:
Uses the forwarder on `partial.x` to automatically capture the `render` call and sparing us the extra block indentation.